packages/@n8n/nodes-langchain/nodes/tools/ToolWorkflow/ToolWorkflow.node.ts

```diff
@@ -314,15 +317,97 @@ export class ToolWorkflow implements INodeType {
 					},
 				],
 			},
+			// ----------------------------------
+			//         Output Parsing
+			// ----------------------------------
+			{
+				displayName: 'Specify Input Schema',
+				name: 'specifyInputSchema',
+				type: 'boolean',
+				noDataExpression: true,
+				default: false,
+			},
```

Comment:
`specifyInputSchema`, `schemaType`, `jsonSchemaExample`, `inputSchema` are duplicated in few places maybe worth extract to common descriptions in `packages/@n8n/nodes-langchain/utils/descriptions.ts`